content/admin/authentication/authenticating-users-for-your-github-enterprise-server-instance/using-ldap.md

```diff
@@ -132,7 +132,7 @@ After you enable LDAP sync, a synchronization job will run at the specified time
 
 {% note %}
 
-**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`.
+**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`. Some variations of Active Directory, such as AD LDS and ADAM, do not have the `userAccountControl` attribute present.
```

Comment:
Thanks for this contribution @affrae !

I'll make one tiny change to the language and :shipit: 

```suggestion
**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`. Some variations of Active Directory, such as AD LDS and ADAM, don't support the `userAccountControl` attribute.
```

Reply:
Sounds great - thank you!